src/neynar-api/neynar-api-client.ts

```diff
@@ -72,6 +72,7 @@ import {
   CastComposerActionsListResponse,
   SubscriptionStatus,
   UserPowerLiteResponse,
+  EmbedType,
```

Comment:
Need a change in OAS.
Please rename this to `EmbedTypes` since its a plural entity